PantryNodeReact/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Chico State Pantry</title>
```

Comment:
I'm not sure who made these changes and why, it is not supposed to be branded as Chico State. Need to remove "Chico State" from the title.

Reply:
changed it to `Food Pantry` for now